.env.baklava

```diff
@@ -114,6 +114,9 @@ EKSPORTISTO_SUFFIX='1'
 ATTESTATION_SERVICE_DOCKER_IMAGE_REPOSITORY="us.gcr.io/celo-testnet/celo-monorepo"
 ATTESTATION_SERVICE_DOCKER_IMAGE_TAG="attestation-service-v1.2.0"
 
+SMS_PROVIDERS=twilio,nexmo
```

Comment:
could also add MessageBird here

Reply:
Prefer to get this in as its been open for a while and add message bird in a separate PR